xbmc/TextureCacheJob.cpp

```diff
@@ -32,7 +32,7 @@
 #include "FileItem.h"
 #include "music/MusicThumbLoader.h"
 #include "music/tags/MusicInfoTag.h"
-#if defined(HAS_OMXPLAYER)
+#if defined(TARGET_RASPBERRY_PI)
```

Comment:
because there are so many defines but they all are just related to the rpi. So we should just have a single define across all the code. 

before we had, `HAS_OMXPLAYER`, `HAVE_OMXLIB`, `HAS_MMAL`, etc. There is no point to have these defines when it only works on the rpi anyway.